app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        issues_found = 0
+        if "vaccine" not in visitor:
+            issues_found += 1
+            raise NotVaccinatedError
+        else:
+            if visitor["vaccine"]["expiration_date"] < date.today():
+                issues_found += 1
+                raise OutdatedVaccineError
+        if "wearing_a_mask" in visitor:
```

Comment:
The check for 'wearing_a_mask' should ensure that the key exists in the visitor dictionary. Consider using `if visitor.get('wearing_a_mask', True) is False:` to handle cases where the key might be missing.